src/site/head.rs

```diff
@@ -0,0 +1,67 @@
+use crate::config::Config;
```

Comment:
i think it would be good to make all these functions private and then wrap them in a public function called "build" that return all the html for the head- thoughts?

Reply:
That was my first thought but unfortunately I got several errors, first I attempted to return all the html inside the head, and it gave me an error because the HTML I was returning didn't have a parent element and then when I added this element I got a new error for axohtml

<img width="878" alt="CleanShot 2023-01-18 at 14 47 45@2x" src="https://user-images.githubusercontent.com/1051509/213187953-99c9dcd4-5675-491d-9475-8f8f35a861c8.png">

Reply:
Error comes from here: https://github.com/axodotdev/axohtml/blob/remove-dodrio/macros/src/html.rs#L31

Reply:
let's file an issue and get this PR merged in the meantime? i'll need more info to debug this properly

Reply:
issue filed https://github.com/axodotdev/oranda/issues/74